Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -2562,10 +2562,15 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
     public func resolve(_ reference: TopicReference, in parent: ResolvedTopicReference, fromSymbolLink isCurrentlyResolvingSymbolLink: Bool = false) -> TopicReferenceResolutionResult {
         switch reference {
         case .unresolved(let unresolvedReference):
+            let referenceCacheIdentifier = ResolvedTopicReference.cacheIdentifier(
```

Comment:
I made this change mainly on a hunch since I noticed that we actually _have_ to re-resolve everything twice. We first resolve relatively links with MarkupRewriter pass and then re-resolve them as absolute links when rendering out to RenderNodes. But you're right, it doesn't seem to make an impact so I'll go ahead and remove it.

I'd be interested in investigating a later (larger) refactoring that removes the MarkupRewriter pass entirely though...